Sources/FlexLayout.swift

```diff
@@ -648,6 +648,73 @@ public final class Flex {
         return self
     }
     
+    /**
+      Set the left and right edges distance from the container edges in pixels.
+      This method is valid only when the item position is absolute (`view.flex.position(.absolute)`)
+      */
+    @discardableResult
+    public func horizontal(_ value: CGFloat) -> Flex {
```

Comment:
Good, could you please add theses methods in the documentation (README.md) in the section https://github.com/layoutBox/FlexLayout#4-absolute-positioning.